### ISSUE TITLE
Filtering by Environment vars

### ISSUE THREAD

**Issue by datacoda** [gregory90/docker-fluent-logentries#1]
Thanks for making this docker container.  I'm in a situation where I don't want to log every container and was thinking of extending this to have some sort of LOG_FILTERS=selecta,selectb to sort through a LOG_FILES environment variable on containers.

Would you like this as a feature, or should I just fork completely and keep the original dockerfile simple.

I'd appreciate any input you may have.
Cheers.

**Comment by datacoda** [comment]
The further I'm getting in, the more I realize I might have to base off phusion/baseimage first.  :(  Out of curiosity, what's your LICENSE for your work?  If I end up forking, I'd need to borrow the logentries type connector - if that would be okay.

**Comment by gregory90** [comment]
I'd really like to have that functionality :) 
Changing base image is all right, but why would you need that?

I've added license and credits - connector is actually based on https://github.com/Woorank/fluent-plugin-logentries 

I'd gladly accept your PR, that's really nice feature to have :)

**Comment by datacoda** [comment]
Hi.  Since I needed to use the alt baseimage, I created a clean slate today at https://github.com/dataferret/docker-logentries  

Primarily, I needed runit to allow docker-gen to reload td-agent whenever the config changed.  In the end, I also ended up using some other fluent plugins to massage the logs into json form.  Eek, it's kinda diverged way more than I expected so I'm leaning towards keeping these two separate.  There's a simplicity with yours that I think is nicer in some circumstances.  

If you still want though, I can build a PR without the json output changes.
